src/main/java/it/gov/pagopa/ranking/service/initiative/ranking/InitiativeOnboardingRankingEndingService.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.ranking.service.initiative.ranking;
+
+import it.gov.pagopa.ranking.model.InitiativeConfig;
+
+import java.util.List;
+
+public interface InitiativeOnboardingRankingEndingService {
```

Comment:
Can I suggest a more simple and focused name
OnboardingRankingRuleEndedService <- if intended to host more op on ended rules
retrieve.OnboardingRankingRuleEndedRetrieverService <- in a specific pakage and with a name focused just on the retrieve operation